logger/logger.go

```diff
@@ -7,31 +7,101 @@ package logger
 import (
 	"fmt"
 	"os"
+	"strings"
 	"time"
 )
 
+var requestedLevel LogLevel
+
+// LogLevel type
+type LogLevel uint32
+
+const (
+	// FatalLevel should be used in fatal situations, the app will exit
+	FatalLevel LogLevel = iota
+
+	// ErrorLevel should be used when someone should really look at the error
+	ErrorLevel
+
+	// InfoLevel should be used during normal operations
+	InfoLevel
+
+	// DebugLevel should be used only during development
+	DebugLevel
+)
+
+// AllLevels exposes all logging levels
+var AllLevels = []LogLevel{
```

Comment:
Why do we need to expose this variable outside the package? Doesn't seems to be used elsewhere.

Reply:
You're right; I'll make it private.